seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumJSEnv.scala

```diff
@@ -101,19 +106,36 @@ object SeleniumJSEnv {
      *  )
      *  }}}
      */
+    def withMaterializeInServer(contentDir: Path, webRoot: URL): Config =
```

Comment:
The Scaladoc attached to this method has an example where the parameters are `String`s. It should be changed to use a `Path` and a `URL` instead.